src/index.tsx

```diff
@@ -1,25 +1,36 @@
 import './index.css';
 import '../node_modules/antd/dist/antd.less';
 import * as React from 'react';
+
 import { render } from 'react-dom';
+
 import { Provider } from 'react-redux';
 import './i18n';
 import { ConfigProvider } from 'antd';
+
 import deDE from 'antd/lib/locale-provider/de_DE';
+
 import { defaults as OlDefaultControls } from 'ol/control';
 import OlMap from 'ol/Map';
-import  { get as OlGetProjection } from 'ol/proj';
-
+import { get as OlGetProjection } from 'ol/proj';
 import OlView from 'ol/View';
 import OlScaleLine from 'ol/control/ScaleLine';
+
 import ProjectionUtil from '@terrestris/ol-util/src/ProjectionUtil/ProjectionUtil';
-import store from './state/store';
-import Main from './Main';
-import SomethingWentWrong from './SomethingWentWrong';
 
 import MapProvider from '@terrestris/react-geo/dist/Provider/MapProvider/MapProvider';
 import { mappify } from '@terrestris/react-geo/dist/HigherOrderComponent/MappifiedComponent/MappifiedComponent';
 
+import {
+  setupStore,
+  loadAppContextStore
+} from './state/store';
+
+import Main from './Main';
+import { SomethingWentWrong } from './SomethingWentWrong';
+
+const MappifiedMain = (mappify)(Main);
```

Comment:
const MappifiedMain = (mappify)(Main);
```suggestion
const MappifiedMain = mappify(Main);
```